_includes/_blogcontent/2024-02-26-userfeedback-2024.md

```diff
@@ -0,0 +1,68 @@
+The virtual gathering of the [Astrophysics Data System Users Group (ADSUG)](https://ui.adsabs.harvard.edu/about/adsug/) on 16-17 November served as a pivotal moment in our ongoing transition from the Astrophysics Data System (ADS) to the [NASA Science Explorer (SciX)](https://scixplorer.org/scixblog/scix). A crucial precursor to this meeting was a user survey, offering a comprehensive view of the user community's perceptions, expectations, and concerns from the 248 survey respondents. 
+The ADSUG meeting delved into these user insights, providing a collaborative space for discussions on maximizing the scientific productivity of the community we serve. The [meeting's agenda](https://ui.adsabs.harvard.edu/about/adsug/past_meetings/2023-11-16-202311-program.html) included recommendations for changes and improvements to services and procedures, reflecting the ADSUG's advisory role.
```

Comment:
Should this be a separate paragraph? It's running together with the previous paragraph in the rendered view - add a blank line between paragraphs to make it a new paragraph (yay markdown)

Oh and same comment as above - make this link a relative one.

Reply:
separated paragraphs and changed to relative link.